Daramola98/WEconnectApp
# Implements Update Business Feature

#### What does this PR do?
This PR enables users to successfully view a business profile.

#### Description of Task to be completed?
Implement View Business profile feature

#### How should this be manually tested?
View application at weconnect-api.herokuapp.com
An authenticated user navigates to his businesses page and clicks update
The user fills the form and submits then should get a success message or errors if any were found

#### What are the relevant pivotal tracker stories?
#156794304
